app/models/user.rb

```diff
@@ -2,5 +2,9 @@ class User < ActiveRecord::Base
   has_many :comments
   has_many :posts
   has_many :likes
-  validates :name, presence: true, length: { minimum: 3, maximum: 50 }
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
```

Comment:
- [ ] The user name must not be blank. Kindly, make sure that you add the validation about it.
`validates :name, presence: true`